oraclizeAPI_0.4.25.sol

```diff
@@ -352,7 +352,7 @@ contract usingOraclize {
 
         if(address(oraclize) != OAR.getAddress())
             oraclize = OraclizeI(OAR.getAddress());
-            oraclizeBytes = OraclizeIBytes(OAR.getAddress());
+            oraclizeBytes = OraclizeIBytes(address(oraclize));
```

Comment:
further can be saved, by pushing the OAR.getAddress() result somewhere on the stack first (save in a variable somewhere before the if statement) and re-using that across all 3 instances of OAR.getAddress here.

Reply:
Including in the coupon modifier that's still around.

Reply:
Good spot, have optimized and push changes to this PR. 

Note that the coupon modifier doesn't exist in the 0.5 API (which is hopefully correct) and in the `0.4.25API` the coupon modifier only calls `OAR.getAddress()` once, so no savings to be gained there.